superchain/configs/mainnet/arena-z.toml

```diff
@@ -36,14 +37,16 @@ data_availability_type = "eth-da"
     scalar = "0x010000000000000000000000000000000000000000000000000c3a30000060a4"
     gasLimit = 30000000
 
-[addresses]
+[roles]
   SystemConfigOwner = "0xBeA2Bc852a160B8547273660E22F4F08C2fa9Bbb"
   ProxyAdminOwner = "0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A"
   Guardian = "0x09f7150D8c019BeF34450d6920f6B3608ceFdAf2"
   Challenger = "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A"
   Proposer = "0x5f16E66D8736B689a430564a31c8d887ca357CD8"
   UnsafeBlockSigner = "0xb774Ca8438319d2a97B9925F4CD248e4C470Ac5B"
   BatchSubmitter = "0x2b8733E8c60A928b19BB7db1D79b918e8E09AC8c"
+
+[addresses]
```

Comment:
Could we use the term `contracts` instead of `addresses` here to disambiguate between this and `roles` (since roles are also addresses)?

Reply:
I'm OK with that.

Reply:
Actually - I think that superchain ops reads the `addresses` struct, so I'd prefer not to change it in this case.